HISTORY.rst

```diff
@@ -2,6 +2,82 @@
 History
 =======
 
+3.0.4 (2025-03-13)
+------------------
+
+This new bugfix release of ``unyt`` fixes bugs discovered since the v3.0.3 release.
+
+* Fix incompatibilities with numpy 2.1 (`PR #512 <https://github.com/yt-
+  project/unyt/pull/512>`_). Thank you to Clément Robert (@neutrinoceros on GitHub)
+  for the contribution.
+
+* Fix stephan-boltzmann constant accuracy and add radiation constant #520,  michael
+  zingale (@zingale on github)add foe and bethe units (`PR #521
+  <https://github.com/yt-project/unyt/pull/521>`_). Thank you to Michael Zingale
+  (@zingale on GitHub)add foe and Bethe units #521, Michael Zingale (@zingale on
+  GitHub) for the contribution.
+
+* Fix raising a unyt array to an array power in sensible cases (`PR #524
```

Comment:
The addition of the bethe and foe limits is technically not a bugfix--we should include it here but call it out as a feature.